### ISSUE TITLE
Click on HTML topic content of node doesn't select the node 

### ISSUE THREAD

**Issue by ScorprocS** [hizzgdev/jsmind#455]
Hi, I don't know if it's really an issue but in my case I would like when we click on html topic content of a node it selects the node.

For those who have the same issue.
Here's my workaround for angular code.

```
 onNodeClick(eventListener: any): void {
      const event: MouseEvent = eventListener as MouseEvent;
      event.preventDefault();
      var target = event.currentTarget as Element;
      let nodeId = target.getAttribute('nodeid');
      if(nodeId){
        this.jm.select_node(nodeId);

      }
    }
```

and add click listener to your html node

`element.addEventListener('click', this.onNodeClick.bind(this));
`

**Comment by hizzgdev** [comment]
Thanks to report the issue. Would you mind sharing the HTML content for reproducing the issue? @ScorprocS

**Comment by ScorprocS** [comment]
Any Html content will not select the node.
I've tried <b>test</b>

**Comment by hizzgdev** [comment]
Thank you, I will investigate it later.

**Comment by hizzgdev** [comment]
The bug has been fixed by #457, and the changes is included in the latest release v0.5.6.
